common/OpTestHMC.py

```diff
@@ -286,7 +286,7 @@ def migrate_lpar(self, src_mg_system=None, dest_mg_system=None):
         self.ssh.run_command(
             'migrlpar -o v -m %s -t %s -p %s' % (src_mg_system, dest_mg_system, self.lpar_name))
         self.ssh.run_command(
-            'migrlpar -o m -m %s -t %s -p %s' % (src_mg_system, dest_mg_system, self.lpar_name))
+            'migrlpar -o m -m %s -t %s -p %s' % (src_mg_system, dest_mg_system, self.lpar_name), timeout=180)
```

Comment:
180 secs timeout is not enough for some setups, please increase to 300 and it works on my setup